src/aiida_workgraph/utils/__init__.py

```diff
@@ -580,3 +580,28 @@ def serialize_socket_data(input_socket: Dict[str, Any]) -> None:
         if isinstance(value, TaggedValue):
             value = value.__wrapped__
         input_socket["property"]["value"] = general_serializer(value)
+
+
+def clean_node_links_manager(data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
def resolve_node_links_managers(data: Dict[str, Any]) -> Dict[str, Any]:
```

You aren't cleaning them (nothing is removed), just resolving them into their `dict` form.